caikit/runtime/http_server/pydantic_wrapper.py

```diff
@@ -55,10 +55,10 @@
 # Base class for pydantic models
 # We want to set the config to forbid extra attributes
 # while instantiating any pydantic models
-# This is done to make sure any oneofs can be
-# correctly infered by pydantic
+# This is done to make sure any one-offs can be
```

Comment:
This is actually not a typo! It's referring to a protobuf `oneof`. The one below definitely is, though.